src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx

```diff
@@ -22,17 +27,118 @@ The New Relic **Explorer** tab is a good place to start gathering information ab
 
 3. In the center pane, click the service you want to know more about, or find your service by entering the name in the top filter bar.
 
-If you need help understanding how we display your data, see the following descriptions of the left-pane options in the UI.
+With the filter bar **Narrow data to...**, you can highlight a specific facet of the telemetry recorded for your service. For example, you may want to see the error rate for a particular version of the service that you've deployed in a canary instance, so you add a filter for `service.version='1.2.3'`.
```

Comment:
What do you see as the relationship between the steps and this info - I almost want a heading 3 or something to transition between the steps and the paragraphs of info. It might just be that I'm not used to a bunch of paragraphs after a step though. Or, it might be that indenting these paragraphs to indicate they're the follow on to the steps is what's most ideal.

Reply:
@mmfred, I switched this up based on SME feedback. So, we now have two sub-sections.